mne/preprocessing/tests/test_maxwell.py

```diff
@@ -970,4 +970,13 @@ def test_triux():
                             st_duration=4., verbose=True)
     assert_meg_snr(sss_py, read_crop(tri_sss_st4_fname), 700., 1600)
 
+
+@testing.requires_testing_data
+def test_MGH_cross_talk():
+    raw = read_crop(raw_fname, (0., 1.))
+    print(raw_fname)
```

Comment:
Thank you guys for guiding me through it, specially for someone who has been spoiled by SVN.